java/ingest/src/test/java/sleeper/ingest/job/IngestJobQueueConsumerIT.java

```diff
@@ -217,8 +212,8 @@ private void consumeAndVerify(Schema sleeperSchema,
         InstanceProperties instanceProperties = getInstanceProperties();
         TableProperties tableProperties = createTable(sleeperSchema);
         TablePropertiesProvider tablePropertiesProvider = new TablePropertiesProvider(AWS_EXTERNAL_RESOURCE.getS3Client(), instanceProperties);
-        StateStoreProvider stateStoreProvider = new StateStoreProvider(AWS_EXTERNAL_RESOURCE.getDynamoDBClient(), new InstanceProperties());
-        StateStore stateStore = stateStoreProvider.getStateStore(tableProperties);
+        StateStore stateStore = new DelegatingStateStore(new InMemoryFileInfoStore(), new FixedPartitionStore(sleeperSchema));
+        StateStoreProvider stateStoreProvider = new FixedStateStoreProvider(tablePropertiesProvider.getTableProperties(TEST_TABLE_NAME), stateStore);
```

Comment:
I've included an extra change in this PR to use an InMemoryStateStore as opposed to the AWS DynamoDB one, this may need to be revisited later down the line when we split this test into IT and unit tests